scripts/release.sh

```diff
@@ -164,7 +199,12 @@ if [[ $? == 1 ]]; then
 
     # This runs Jazzy to generate the documentation.
     echo "Running Jazzy to generate documentation..."
-    jazzy --clean --objc --framework-root SmartDeviceLink --sdk iphonesimulator --umbrella-header SmartDeviceLink/public/SmartDeviceLink.h --theme theme --output docs
+    # TODO - I do not know if this will work as intended.
```

Comment:
I'll have to test this on my M1 to ensure it works, then we should remove this TODO.